ezra-project.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Version=1.0
+Type=Application
+Name=Ezra Project
+Comment=ezra-project
+Exec=ezra-project
+Icon=ezra-project
+Terminal=false
+StartupNotify=false
+Categories=Application;
```

Comment:
In the [other package configuration](https://github.com/tobias-klein/ezra-project/tree/master/package_config) files I've been using the following categories:
```
  "categories": [
    "Education",
    "Utility",
    "Spirituality"
  ],
```
Shouldn't we use the same here?

Reply:
Yes, probably, but we should find a map of categories because those sound like Ubuntu app store categories rather that FSF desktop categories.

Reply:
These categories are actually the ones from freedesktop.org.

Education and Utility from the [main categories](https://specifications.freedesktop.org/menu-spec/latest/apa.html) and Spirituality from the [additional categories](https://specifications.freedesktop.org/menu-spec/latest/apas02.html) on specifications.freedesktop.org.

I'm using these categories consistently both for *.deb packages and *.rpm packages.

Reply:
Here is the 1.0 spec categories: https://specifications.freedesktop.org/menu-spec/menu-spec-1.0.html#category-registry

* "Spirituality" is not a thing.
* Your use of "Utility" is probably incorrect, this is not a utility in the software sense.
* I think "Education" is the only main category that makes sense.
* For secondary categories "DataVisualization" and "Liturature" might make sense.

Reply:
Oops, my web page search foo failed _and_ I missed your comment because I was typing mine. I'll update....